packages/KPop/KPop.vue

```diff
@@ -281,7 +281,10 @@ export default {
           this.createInstance()
         }
       } else if (this.$refs.popper && this.$refs.popper.contains(e.target) && this.onPopoverClick) {
-        this.onPopoverClick()
+        const shouldShow = !this.onPopoverClick()
```

Comment:
I dont understand this line. So is `onPopoverClick()` assumed to return a boolean? cause we always know it's defined at this section of the code.

Reply:
Here's how the typescript prop would look
```ts
onPopoverClick: {
  type: Function as PropType<() => boolean|undefined>,
  required: false,
  default: () => undefined
}
```